code/modules/codex/entries/paperwork.dm

```diff
@@ -22,7 +22,8 @@
 \[hr\] : Adds a horizontal rule.<br>
 \[list\] - \[/list\] : A list. (Does not work with crayons)<br>
 \[*\] : A dot used for lists. (Does not work with crayons)<br>
-\[logo\] - Inserts EXO logo image.<br>
+\[logo\] - Inserts regular CEC logo image.<br>
+\[ceclogo\] - Inserts the formal CEC Logo image. This logo is only used for official, outbound or external documents. Not commonly used internally.<br>
```

Comment:
help text for formal logo is here

fixing it would be best, but if not, remove this text for now

Reply:
Fixed.